apis/apiextensions/v1alpha1/mrd_types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v2alpha1
+package v1alpha1
```

Comment:
just a note to not forget this pending update to the MRD type in its current location 🤓 : https://github.com/crossplane/crossplane/pull/6660/files#diff-d36410d1934b7e5180cc12388a455f4aaf63ef37eeeb443b467b8933dd7be31f